### PR TITLE
add: FAQ section

### PR DESCRIPTION
**Pull request by admantiumblack**

### CODE REVIEW

File: app.py [hunk 1]
```diff
@@ -81,10 +81,68 @@ def speak(text: str):
...
+            f'<div class="faq-question">{item["question"]}</div>', unsafe_allow_html=True)
+        st.markdown(
+            f'<div class="faq-answer">{item["answer"]}</div>', unsafe_allow_html=True)
+
+
 def main():
     st.markdown("<br>", unsafe_allow_html=True)
     st.title("FinNetra")
-    st.write("Menyediakan tunanetra hak atas akses finansial yang setara melalui kekuatan AI dan ML.")
+    st.write(
```
Comment (elvanselvano71) [inline app.py:116]: Unnecessary line changes

File: app.py [hunk 2]
```diff
@@ -81,10 +81,68 @@ def speak(text: str):
...
+            font-size: 16px;
+            color: white;
+            margin-bottom: 20px;
+            text-align: center;
+        }
+        </style>
+        """, unsafe_allow_html=True)
+
+    st.title("Frequently Asked Questions (FAQ)")
+    faq_items = [
```
Comment (elvanselvano71) [inline app.py]: Store in constants folder, save as json file

File: app.py [hunk 3]
```diff
@@ -81,10 +81,68 @@ def speak(text: str):
...
+        {
+            "question": "Apa itu Finnetra?",
+            "answer": "Finnetra adalah aplikasi berbasis kecerdasan buatan (AI) dan machine learning (ML) yang dikembangkan untuk memperluas inklusi keuangan bagi orang dengan disabilitas visual. Finnetra dikembangkan oleh Tim HaloBI, yang terdiri dari Elvan Selvano, Roland, Novita Furia Putri, dan Dody Harianto. Proyek bertujuan untuk memberikan akses keuangan yang lebih mudah bagi tunanetra."
+        },
+        {
+            "question": "Mengapa kami membuat Finnetra?",
+            "answer": "Kami menciptakan Finnetra sebagai respons terhadap kebutuhan yang belum terpenuhi bagi orang dengan disabilitas visual untuk mengakses layanan keuangan secara mandiri. Kami ingin membantu mengatasi hambatan-hambatan ini melalui teknologi AI sehingga dapat memberikan jawaban kepada tunanetra dengan latar belakang berbeda-beda."
```
Comment (elvanselvano71) [inline app.py]:
```suggestion
            "answer": "Kami menciptakan Finnetra sebagai respon terhadap kebutuhan yang belum terpenuhi bagi orang dengan disabilitas visual untuk mengakses layanan keuangan secara mandiri. Kami ingin membantu mengatasi hambatan-hambatan ini melalui teknologi AI sehingga dapat memberikan jawaban kepada tunanetra dengan latar belakang berbeda-beda."
```

File: app.py [hunk 4]
```diff
@@ -81,10 +81,68 @@ def speak(text: str):
...
+        },
+        {
+            "question": "Mengapa tunanetra perlu menggunakan aplikasi kami?",
+            "answer": "Finnetra didesain cukup ramah untuk tunanetra karena hanya memerlukan suara pengguna dan dapat memberikan dampak yang signifikan bagi mereka. Dengan aplikasi ini, mereka dapat melakukan transaksi keuangan, mengelola tabungan, dan mendapatkan rekomendasi layanan keuangan yang sesuai dengan kebutuhan mereka, yang semuanya dapat meningkatkan kemandirian dan kualitas hidup pengguna."
```
Comment (elvanselvano71) [inline app.py]:
```suggestion
            "answer": "Finnetra dirancang ramah bagi tunanetra karena hanya memerlukan suara pengguna. Melalui aplikasi ini, pengguna dapat menerima rekomendasi layanan keuangan yang dipersonalisasi sesuai kebutuhan spesifik masing-masing, menghadirkan solusi keuangan yang inovatif."
```

File: app.py [hunk 5]
```diff
@@ -81,10 +81,68 @@ def speak(text: str):
...
+        },
+        {
+            "question": "Bagaimana cara menggunakan Finnetra?",
+            "answer": "Finnetra dapat diakses melalui web baik menggunakan telepon genggam atau laptop. Dengan demikian, aksesnya sangat mudah bagi siapa pun yang membutuhkan layanan kami."
```
Comment (elvanselvano71) [inline app.py]:
```suggestion
            "answer": "Finnetra dapat diakses melalui web, baik menggunakan telepon genggam maupun laptop. Dengan demikian, aksesnya menjadi sangat mudah dan praktis bagi siapa pun yang memerlukan layanan kami."
```

File: app.py [hunk 6]
```diff
@@ -81,10 +81,68 @@ def speak(text: str):
...
+        },
+        {
+            "question": "Apakah ada biaya yang perlu saya keluarkan untuk menggunakan Finnetra sebagai pengguna?",
+            "answer": "Tidak, Finnetra dapat digunakan secara gratis oleh pengguna. Kami ingin memastikan bahwa layanan kami dapat diakses oleh semua orang tanpa hambatan finansial."
```
Comment (elvanselvano71) [inline app.py]:
```suggestion
            "answer": "Tidak, Finnetra dapat digunakan secara gratis oleh pengguna. Kami berkomitmen untuk memastikan bahwa layanan kami dapat diakses oleh semua orang tanpa hambatan finansial."
```

File: app.py [hunk 7]
```diff
@@ -81,10 +81,68 @@ def speak(text: str):
...
+        },
+        {
+            "question": "Model AI / machine learning apa yang digunakan oleh Finnetra?",
+            "answer": "Finnetra menggunakan model OpenAI Whisper untuk merekam suara pengguna dan mengubah suara menjadi teks, model GPT untuk memproses data keuangan pengguna dan memberikan rekomendasi layanan keuangan yang relevan untuk pengguna, dan Google Text to Speech (gTTS) yang digunakan untuk mengubah hasil rekomendasi menjadi audio sehingga dapat diterima oleh tunanetra."
```
Comment (elvanselvano71) [inline app.py]:
```suggestion
            "answer": "Finnetra menggunakan model OpenAI Whisper untuk merekam dan mengubah suara pengguna menjadi teks, serta Large Language Model (LLM) untuk memproses data keuangan pengguna dan memberikan rekomendasi layanan keuangan yang relevan. Selain itu, Google Text to Speech (gTTS) digunakan untuk mengubah hasil rekomendasi menjadi audio, sehingga dapat diterima dengan mudah oleh tunanetra."
```

File: app.py [hunk 8]
```diff
@@ -81,10 +81,68 @@ def speak(text: str):
...
+            text-align: center;
+        }
+        </style>
+        """, unsafe_allow_html=True)
+
+    st.title("Frequently Asked Questions (FAQ)")
+    faq_items = [
+        {
+            "question": "Apa itu Finnetra?",
+            "answer": "Finnetra adalah aplikasi berbasis kecerdasan buatan (AI) dan machine learning (ML) yang dikembangkan untuk memperluas inklusi keuangan bagi orang dengan disabilitas visual. Finnetra dikembangkan oleh Tim HaloBI, yang terdiri dari Elvan Selvano, Roland, Novita Furia Putri, dan Dody Harianto. Proyek bertujuan untuk memberikan akses keuangan yang lebih mudah bagi tunanetra."
```
Comment (elvanselvano71) [inline app.py]:
```suggestion
            "answer": "Finnetra adalah aplikasi berbasis kecerdasan buatan (AI) dan machine learning (ML) yang dirancang untuk memperluas inklusi keuangan bagi individu dengan disabilitas visual. Dikembangkan oleh Tim HaloBI, yang terdiri dari Elvan Selvano, Roland, Novita Furia Putri, dan Dody Harianto, proyek ini bertujuan untuk memberikan akses keuangan yang lebih mudah dan efisien bagi tunanetra."
```

File: app.py [hunk 9]
```diff
@@ -81,10 +81,68 @@ def speak(text: str):
...
+            "answer": "Tidak, Finnetra dapat digunakan secara gratis oleh pengguna. Kami ingin memastikan bahwa layanan kami dapat diakses oleh semua orang tanpa hambatan finansial."
+        },
+        {
+            "question": "Model AI / machine learning apa yang digunakan oleh Finnetra?",
+            "answer": "Finnetra menggunakan model OpenAI Whisper untuk merekam suara pengguna dan mengubah suara menjadi teks, model GPT untuk memproses data keuangan pengguna dan memberikan rekomendasi layanan keuangan yang relevan untuk pengguna, dan Google Text to Speech (gTTS) yang digunakan untuk mengubah hasil rekomendasi menjadi audio sehingga dapat diterima oleh tunanetra."
+        },
+        {
+            "question": "Jika saya memiliki masukan atau tidak dapat menggunakan Finnetra, kepada siapa saya harus menghubungi?",
+            "answer": "Jika Anda memiliki masukan atau mengalami kesulitan dalam menggunakan Finnetra, Anda dapat mengacu pada tautan https://github.com/elvanselvano/streamlit-whisper atau menghubungi tim kami melalui informasi kontak yang tersedia di dalam tautan tersebut. Kami siap membantu Anda dengan segala pertanyaan atau masalah yang Anda hadapi."
```
Comment (elvanselvano71) [inline app.py]:
```suggestion
            "answer": "Jika Anda memiliki masukan atau mengalami kesulitan dalam menggunakan Finnetra, Anda dapat mengunjungi tautan https://github.com/elvanselvano/streamlit-whisper atau menghubungi tim kami melalui informasi kontak yang tersedia di sana. Kami siap membantu Anda dengan segala pertanyaan atau masalah yang mungkin Anda hadapi."
```

File: app.py (hunk 1 above)
Comment (admantiumblack) [reply app.py:116]: auto formater

**Review by elvanselvano71 — APPROVED** [review]

**Pull request merged**